schema.go

```diff
@@ -106,3 +110,71 @@ func findBuildByBuilder(db *mgo.Database, builder string, buildnum int) (*LogKee
 	}
 	return build, nil
 }
+
+func findAndDeleteTests(db *mgo.Database) ([]Test, error) {
+	now := time.Now()
+	query := bson.M{
+		"started": bson.M{"$lte": now.Add(-deletePassedTestCutoff)},
+		"failed":  false,
+	}
+	tests := []Test{}
+
+	err := db.C("tests").Find(query).Sort("started").Select(bson.M{"_id": 1}).All(&tests)
```

Comment:
Per a comment from john, will test a new version of this that limits the number of tests returned to some constant since logkeeper is absurdly huge.